internal/utils/marshall.go

```diff
@@ -62,3 +38,24 @@ func UnmarshallList(arg string) ([]string, error) {
 	}
 	return res, nil
 }
+
+func MarshallToString(args interface{}) (string, error) {
+	jsonData, err := json.Marshal(args)
+	if err != nil {
+		return "", fmt.Errorf("json.Marshal: %w", err)
+	}
+
+	encoded := base64.StdEncoding.EncodeToString(jsonData)
+	if err != nil {
+		return "", fmt.Errorf("base64.StdEncoding.EncodeString: %w", err)
+	}
+	return encoded, nil
+}
+
+func MarshallToBytes(args interface{}) ([]byte, error) {
+	encoded, err := MarshallToString(args)
+	if err != nil {
+		return []byte{}, nil
```

Comment:
Did you want to return `nil, err` here?

Reply:
probably. I'll make the change. Thank you!